libs/features/pages/consultancy-page/src/lib/Component/elewa-website-learning-services/elewa-website-learning-services.component.scss

```diff
@@ -0,0 +1,43 @@
+.learning-services {
+  height: 100vh;
+  display: flex;
+  justify-content: center; /* Center horizontally */
+  align-items: center; /* Center vertically */
+  height: 100vh; 
+    
+
+  
+    .learning-services-container {
+      display: grid;
+      grid-template-columns: repeat(2, 1fr); /* Two equal-width columns */
+      grid-gap: 10px;
+      padding: 0 100px 100px 100px;/* Adjust the gap between items as needed */
+
+     
+  
+      /* Styling for the info cards within ngFor loop */
+      .info-card {
+        /* Common styles for info cards */
+        padding: 20px;
+        
+  
+      }
+    }
+  }
+  
+/* Styling for the <h6> element */
+h1 {
+  text-align: center;
+  font-size: 52px;
+  font: normal normal medium 62px/80px DM Sans;
```

Comment:
For reuse of fonts, use CSS variables specified in elements/theming/_variables.scss for specifying font family